src/people.rs

```diff
@@ -25,9 +26,9 @@ define_data_plugin!(
 
 // Represents a unique person - the id refers to that person's index in the range
 // 0 to population - 1 in the PeopleData container.
-#[derive(Clone, Copy, Hash, PartialEq, Eq)]
+#[derive(Clone, Copy, Hash, PartialEq, Eq, Serialize, Deserialize)]
 pub struct PersonId {
-    id: usize,
+    pub id: usize,
```

Comment:
Do you still need to make this public if you have get_person_id?

Reply:
Well, I am not sure. I tried making it private, but changes in person properties emit events that contain a `person_id` (of type PersonId). This is causing issues in the incidence report as well. To send the report, I am doing the following (which doesn't look great):
```rust
`context.send_report(IncidenceReportItem` {
  time: context.get_current_time(),
  person_id: event.person_id.id
}
```
Should we implement something like `get_id()` for PersonId, so that we could run `event.person_id.get_id()`? @k88hudson-cfa

Reply:
Oh yeah, the reporting use case seems important – I think it's ok just to leave it as `pub` then

Reply:
Could you change/use the implementation for `Display` instead?

Reply:
Merged your implementation